encrypted-media-respec.html

```diff
@@ -2547,7 +2557,12 @@
 
         <dt>attribute EventHandler onencrypted</dt>
         <dd>
-          <p>Event handler for the <a def-id="encrypted"></a> event MUST be supported by all HTMLMediaElements as both a content attribute and an IDL attribute.</p>
```

Comment:
See also https://www.w3.org/Bugs/Public/show_bug.cgi?id=23897 where the IDL does not imply markup support.

Note: This discussion only applies to the `HTMLMediaElement` handlers.